Start.tsx

```diff
@@ -187,12 +205,12 @@ function Start() {
       <ErrorBoundary onError={logError} FallbackComponent={MemmyErrorView}>
         {/* eslint-disable-next-line react/style-prop-object */}
         <StatusBar style={statusBarColor} />
-        <GestureHandlerRootView style={{ flex: 1 }}>
+        <GestureHandlerRootView style={{ flex: 1, backgroundColor: selectedTheme.colors.app.bg }}>
           <ActionSheetProvider>
-            <>
+            <View style={{ flex: 1, backgroundColor: selectedTheme.colors.app.bg }}>
```

Comment:
This change might no longer be needed. When I get another break I'll try taking it out and see if things still look good.

Reply:
Cool. I'll just wait for you to check on that, everything else looks fine.

Reply:
This wasn't needed. Removed.